event-sourcing/event-repository/event-repository-jdbc/src/test/java/uk/gov/justice/services/eventsourcing/repository/jdbc/JdbcBasedEventRepositoryTest.java

```diff
@@ -225,11 +225,11 @@ public void shouldGetActiveStreamOfStreamIds() throws Exception {
         final UUID streamId2 = UUID.fromString("4b4e80a0-76f7-476c-b75b-527e38fb252e");
         final UUID streamId3 = UUID.fromString("4b4e80a0-76f7-476c-b75b-527e38fb253e");
```

Comment:
Is this a new test or just an old test with the method calls changed? If it is then we should really have a new test